packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx

```diff
@@ -64,6 +70,7 @@ const FooterWrapper = styled(FlyoutWrapper)`
   justify-content: flex-end;
   display: flex;
   flex-direction: column;
+  margin-top: -16px;
```

Comment:
Nothing wrong with this, but one thing I've been doing lately is using `transform: translateY(-16px)` as it won't affect the elements around the item you're moving.